src/components/lib/Pages.tsx

```diff
@@ -31,7 +31,7 @@ export abstract class Pages {
         reason += `You are ${props.form.age?.years} years `;
         reason += `and ${props.form.age?.months} months old. `;
       }
-      Object.keys(result.match.answers).forEach((id) => {
+      Object.keys(result.match.responses).forEach((id) => {
```

Comment:
Naming is hard. `responses` makes me thing of network responses not answers. But this is just my opinion... feel free to ignore it.